receiver/aerospikereceiver/metadata.yaml

```diff
@@ -0,0 +1,125 @@
+name: aerospikereceiver
+
+resource_attributes:
+  node_name:
+    description: Name of the Aerospike node collected from
+    type: string
+  namespace:
+    description: Name of the Aerospike namespace
+    type: string
+
+attributes:
```

Comment:
A lot of these attribute names could probably have more succinct values, which can be configured using the `value` key, e.g.
```yaml
scan_type:
    value: type
    description: Type of scan operation performed on a namespace
    enum:
      - aggr
      - basic
      - ops_bg
      - udf_bg
```

I think we generally try to keep the value short like this, as long as they can be understood from context.